infra/modules/database/main.tf

```diff
@@ -6,7 +6,6 @@ locals {
   primary_instance_name = "${var.name}-primary"
   role_manager_name     = "${var.name}-role-manager"
   role_manager_package  = "${path.root}/role_manager.zip"
-  name                  = substr(var.name, 0, 12)
```

Comment:
linking for cross-reference https://github.com/navapbc/template-infra/pull/469/files#r1380877494